File: sudoku-common/common-core/src/main/java/com/zumin/sudoku/common/core/config/CommonCoreProperties.java
package com.zumin.sudoku.common.core.config;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Data
@Component
@ConfigurationProperties(prefix = "common.core")
public class CommonCoreProperties {

  /**
   * 项目包名
   */
  private String projectPackage = "com.zumin.sudoku";
}

File: sudoku-common/common-core/src/main/java/com/zumin/sudoku/common/core/CommonCoreProperties.kt
package com.zumin.sudoku.common.core

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding

@ConstructorBinding
@ConfigurationProperties("common.core")
data class CommonCoreProperties(
  // 项目包名
  val projectPackage: String,
)
